docker-compose.yml

```diff
@@ -0,0 +1,13 @@
+version: "3.8"
+services:
+    api:
+        build: ./server
+        ports:
+            - "5000:5000"
+    www:
+        build:
+            context: ./client
+            args:
+                HERE_API: ${HERE_MAPS_TOKEN}
+        ports:
+            - "80:80"
```

Comment:
port 80 wasn't working on my machine, So changed it to 8080:80

Reply:
In the instructions in the readme I mentioned ensuring port 80 is open. Do you think this is enough or should I add something more?